docs/superpowers/plans/2026-05-19-cv-template-classic.md

```
@@ -0,0 +1,539 @@
+**`{{EDUCATION}}` — one block per degree:**
+```html
+<div class="edu-item">
+  <div class="edu-header">
+    <span class="edu-org">Visvesvaraya Technological University</span>
+    <span class="edu-year">Aug 2017 - Aug 2021</span>
+  </div>
+  <div class="edu-degree">B.E, Computer Science and Engineering</div>
+  <div class="edu-gpa">CGPA: 8.0</div>
+</div>
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)